components/camel-sip/src/main/java/org/apache/camel/component/sip/listener/SipPresenceAgentListener.java

```diff
@@ -85,21 +93,21 @@ private void sendNotification(EventHeader eventHeader, boolean isInitial, Object
         ((SipURI)sipPresenceAgent.getConfiguration().getContactHeader().getAddress().getURI()).setParameter(
                 sipPresenceAgent.getConfiguration().getFromUser(), sipPresenceAgent.getConfiguration().getFromHost());
 
-        SubscriptionStateHeader sstate;
+        SubscriptionStateHeader state;
```

Comment:
@nikvaessen the original name, ```sstate``` appears to be an abbreviation of Subscription State, not a typo.  Could you rename this variable back to ```sstate```?